examples/example_data_collection_vx300s.py

```diff
@@ -57,6 +57,11 @@ def main(args):
         nc.log_language("Pick up the cube and pass it to the other robot", timestamp=t)
         nc.log_rgb(CAM_NAME, obs.cameras[CAM_NAME].rgb, timestamp=t)
 
+        if obs.end_effector_poses is not None:
+            nc.log_end_effector_poses(obs.end_effector_poses, timestamp=t)
+        if obs.gripper_open_amounts is not None:
+            nc.log_parallel_gripper_open_amounts(obs.gripper_open_amounts, timestamp=t)
```

Comment:
To keep the example clean, remove the ifs. We know that obs.end_effector_poses will not be none, same with gripper open amounts